### PR TITLE
Added dockerfile and docker readme.

### PR DESCRIPTION
**Pull request by martolini**
Adding a `Dockerfile` and instructions on how to run it out of the box with docker.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,18 @@
 
 `yarn` followed by `yarn build` should work on systems with a shell.
 
+## Run with docker
```
Comment (wjoel) [inline README.md]: Docker instead of docker

File: README.md [hunk 2]
```diff
@@ -7,6 +7,18 @@
...
+
+Building the image will build the plugin and copy it into the grafana/plugins folder.
+
+`$ docker build -t cognite-grafana .`
+
+`$ docker run -d --name grafana -p 3000:3000 cognite-grafana`
+
+Now you can access it at http://localhost:3000
+
+Standard username/password for logging in is admin/admin. See http://docs.grafana.org/installation/docker/ for configuration details.
```
Comment (wjoel) [inline README.md]: Add a newline somewhere, at ~70-80 characters.

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,13 @@
+FROM node:9 as builder
+
+# Builds cognite-grafana-datasource
+WORKDIR /app
+
+COPY . /app
+RUN yarn
+RUN yarn build
+
+FROM grafana/grafana
```
Comment (wjoel) [inline Dockerfile]: Add a tag, `:5.2.3` since we don't have guarantees about backwards compatibility for plugins afaik

File: README.md [hunk 4]
```diff
@@ -7,6 +7,18 @@
...
 
+## Run with docker
+
+Building the image will build the plugin and copy it into the grafana/plugins folder.
+
+`$ docker build -t cognite-grafana .`
+
+`$ docker run -d --name grafana -p 3000:3000 cognite-grafana`
+
+Now you can access it at http://localhost:3000
```
Comment (wjoel) [inline README.md]: Make this a link?

File: README.md (hunk 1 above)
Comment (martincognite) [reply README.md]: Done

File: Dockerfile (hunk 3 above)
Comment (martincognite) [reply Dockerfile]: Good point

File: README.md (hunk 4 above)
Comment (martincognite) [reply README.md]: Github will parse that as a link by default

**Pull request merged**